homework-solutions/ts/hw-14/task-1.ts

```diff
@@ -0,0 +1,38 @@
+
+//1. Создайте дженерик функцию wrapInArray, которая принимает значение любого типа и возвращает его в виде массива этого типа.
+    function wrapInArray<T>(result: T):[T]{
```

Comment:
```suggestion
    function wrapInArray<T>(result: T):[T]{
```
 Формально ответ правильный, просто по приколам TS у тебя получился Tuple, а не массив. Потому исходя из задачи лучше T[]

Reply:
Done